api/app/signals/apps/api/templatetags/extra_properties_tags.py

```diff
@@ -1,8 +1,23 @@
 from django import template
+from django.conf import settings
+
+from signals.apps.api.ml_tool.utils import url_from_category
 
 register = template.Library()
 
 
 @register.filter
 def is_a_list(value):
     return isinstance(value, (list, tuple))
+
+
+@register.filter
+def get_extra_properties(signal):
+    if not settings.FEATURE_FLAGS.get('API_FILTER_EXTRA_PROPERTIES', False):
+        return signal.extra_properties
+
+    category_urls = [url_from_category(signal.category_assignment.category), ]
+    if signal.category_assignment.category.is_child():
+        category_urls.append(url_from_category(signal.category_assignment.category.parent))
+
+    return filter(lambda x: 'category_url' in x and x['category_url'] in category_urls, signal.extra_properties)
```

Comment:
Are we sure that we never have more than the path to a category in `category_url`? Also how  is the optional final `/` treated? If we get these things wrong `'category_url' in x` will fail.

Reply:
This is already implemented in the serializer on extra_properties in the exact same way. So I suggest letting this the way it is for now and test this thoroughly on ACC. If it seems to be an issue we will fix it.